code/Mermaid

```diff
@@ -0,0 +1,282 @@
+```mermaid
```

Comment:
The file should probably be placed somewhere else -- putting it in your personal folder for now is probably fine (?). I think you can also name the file something like `chunks.md` since this is a Markdown file.